src/Styleguide/Components/FollowArtistPopover/state.ts

```diff
@@ -0,0 +1,22 @@
+import { Container } from "unstated"
+
+interface State {
+  excludeArtistIds?: string[]
```

Comment:
Was also wondering in my trying to build this if this property should be `[String]` type since that is expected from the MP side. Not sure I understand the difference.

Reply:
ah yea, so the object and concept is identical ('a list of strings/numbers/whatever'), but the type systems are different and so is the syntax.

Typescript- `string[]` or `Array<string>`
GraphQL- `[String]`